chainer/training/extensions/print_report.py

```diff
@@ -24,10 +24,19 @@ class PrintReport(extension.Extension):
     """
 
     def __init__(self, entries, log_report='LogReport', out=sys.stdout):
+        if not hasattr(out, "write"):
```

Comment:
Please remove this check.

Reply:
Sorry I've simply overlooked this comment. Now fixed.